demo/src/screens/componentScreens/PickerScreen.tsx

```diff
@@ -107,7 +118,12 @@ export default class PickerScreen extends Component {
     filter: undefined,
     statOption: [],
     scheme: undefined,
-    contact: 0
+    contact: 0,
+    selectAllType: 'button' as 'button' | 'checkbox' | undefined
```

Comment:
This should probably be `'button' as PickerSelectionStatusProps['selectAllType']` (`PickerSelectionStatusProps` should be exported`)